### PR TITLE
Remove fomantic `text` module

### PR DESCRIPTION
**Pull request by delvh**
Corollary to #26775:
All selectors I found that are actually used and not necessarily present in the current code have been copied to `web_src/css/base.css`.
Everything else should be a clean removal.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
We can potentially drop the `!important` from the `.text.red` classes etc, but it might have a side-effect.

**Comment by delvh** [comment]
Let's leave that for follow-up PRs…

**Comment by silverwind** [comment]
It'll likely be no issue, 2 classes is heavier than many things. But yes, we can do this another time, let's not forget it.

File: web_src/css/base.css [hunk 1]
```diff
@@ -1175,6 +1175,14 @@ img.ui.avatar,
   color: var(--color-gold) !important;
 }
 
+.text.disabled {
+  opacity: var(--opacity-disabled);
+}
+
+.text.small {
+  font-size: 0.75em;
+}
```
Comment (wxiaoguang) [inline web_src/css/base.css:1180]: Please remove line 1260

image

File: web_src/css/base.css [hunk 2]
```diff
@@ -1175,6 +1175,14 @@ img.ui.avatar,
   color: var(--color-gold) !important;
 }
 
+.text.disabled {
+  opacity: var(--opacity-disabled);
+}
```
Comment (wxiaoguang) [inline web_src/css/base.css]: I am pretty sure this is a no-op, it could be removed.

File: web_src/fomantic/build/semantic.css [hunk 3]
```diff
@@ -19897,138 +19897,3 @@ input::selection {
 /*******************************
          Site Overrides
 *******************************/
-/*!
```
Comment (wxiaoguang) [inline web_src/fomantic/build/semantic.css:19375]: Did you remove it manually? I think it needs `make fomantic`, and the generated file doesn't have the last new line.

**Comment by delvh** [comment]
@wxiaoguang please re-review.

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**